pkg/helper/client.go

```diff
@@ -148,8 +147,18 @@ func ApplyDesiredState(nodeNetworkState *nmstatev1alpha1.NodeNetworkState) (stri
 	return setOutput + outputVlanFiltering, nil
 }
 
-func filterOut(currentState nmstatev1alpha1.State, interfacesFilter string) (nmstatev1alpha1.State, error) {
-	if interfacesFilter == "" {
+func getFilter() {
+	if !globCompiledIsSet {
+		interfacesFilter := os.Getenv("INTERFACES_FILTER")
+		globCompiled = glob.MustCompile(interfacesFilter)
+		globCompiledIsSet = true
+	}
+}
+
+func filterOut(currentState nmstatev1alpha1.State) (nmstatev1alpha1.State, error) {
+	getFilter()
+
+	if globCompiled.Match("") {
```

Comment:
We should compare with glob.Nothing (or what was it). That should be less expensive (if it works)

Reply:
`match.Nothing` isn't expression so it can't be compared with the compiled glob.
So sticking with empty match for now.